ecl/wutest/wutest.cpp

```diff
@@ -1511,6 +1511,18 @@ class WuTest : public CppUnit::TestFixture
         Owned<IWorkUnit> global = factory->getGlobalWorkUnit(NULL, NULL);
         ASSERT(global != NULL);
         ASSERT(streq(global->queryWuid(), "global"));
+
+        Owned<IWUResult> result = global->updateGlobalByName("Result 1");
+        result->setResultScalar(true);
+        result->setResultInt(53);
+        result.clear();
+        global.clear();
+
+        Owned<IConstWorkUnit> wu = factory->getGlobalWorkUnit(NULL, NULL);
```

Comment:
It would also be sensible to test getting a global result from an arbitrary workunit - using ResultSequencePersist.